yazi-scheduler/src/scheduler.rs

```diff
@@ -174,7 +174,8 @@ impl Scheduler {
 		let file = self.file.clone();
 		_ = self.micro.try_send(
 			async move {
-				file.trash(FileOpTrash { id, target, length: 0 }).await.ok();
+				file.trash(FileOpTrash { id, target: target.clone(), length: 0 }).await.ok();
+				Pump::push_delete(target);
```

Comment:
I don't really like adding `trash` to `delete` because they're two different concepts, I think it deserves a new `trash` event kind.

Reply:
Ok, I'll add a new event kind. It should be much simpler after the latest changes.

Reply:
Ok it's done now and seems to work 
![image](https://github.com/sxyazi/yazi/assets/300791/e90d4682-5590-4de2-8674-5f438576823e)